libyul/AST.h

```diff
@@ -35,54 +35,62 @@ namespace solidity::yul
 
 using Type = YulString;
 
-struct TypedName { langutil::SourceLocation location; YulString name; Type type; };
+struct DebugData
+{
+	explicit DebugData(langutil::SourceLocation _yulLocation = {}, langutil::SourceLocation _sollLocation = {}): irLocation(std::move(_yulLocation)), originLocation(std::move(_sollLocation))
+	{}
+	langutil::SourceLocation irLocation;
```

Comment:
Oh, perhaps this explains my confusion. I do not think this should be called `irLocation`, just kept as `location`.